kubemarine/core/group.py

```diff
@@ -630,7 +630,8 @@ def wait_and_get_active_nodes(self, timeout=None) -> 'NodeGroup':
     def _await_rebooted_nodes(self, timeout=None, initial_boot_history: NodeGroupResult = None) -> _HostToResult:
 
         if timeout is None:
-            timeout = self.cluster.globals['nodes']['boot']['defaults']['timeout']
+            # boot_timeout is set for all nodes, so we can take any node's boot_timeout
+            timeout = self.cluster.inventory['nodes'][0].get('boot_timeout', self.cluster.globals['nodes']['boot']['defaults']['timeout'])
```

Comment:
New section `globals: {}` is added to cluster.yaml with the same structure as in globals.yaml for node readiness and reboot timeouts. It overrides values defined in globals.yaml for allowed parameters.

Parameter `timeout_download` can also be moved in `globals:`, but in a separate PR because it is out of scope of this PR.

Reply:
Issue on `timeout_download`: https://github.com/Netcracker/KubeMarine/issues/374